modules/openapi-generator/src/main/resources/JavaJaxRS/resteasy/pojo.mustache

```diff
@@ -1,4 +1,6 @@
 import io.swagger.annotations.*;
+{{#useBeanValidation}}import javax.validation.Valid;{{/useBeanValidation}}
+
```

Comment:
@tnmtechnologies maybe we don't need another blank line here?

Reply:
You are right, no problem to remove the blank line.